packages/core/src/uma.ts

```diff
@@ -487,6 +490,31 @@ export async function getPayReqResponse({
   };
 }
 
+async function getSignedCompliancePayeeData(
+  receivingVaspPrivateKeyBytes: Uint8Array,
+  payerIdentifier: string | undefined,
+  payeeIdentifier: string | undefined,
```

Comment:
Should these actually be required fields instead? It seems like they're needed for the signature, right?

Reply:
hm currently the identifiers are optional in payer/payee data - I'm not sure if this was intentional?
if so, the signature is okay to fallback to undefined -- the unit test actually covers this!

Reply:
hmm so in 0.x payerdata requires identifier and I think that needs to stay the case for uma 1.0 since it's also used in other places (the payreq signature).  It looks like I missed that in the `getSignablePayRequestPayload`, but we should make that explicit. Otherwise the counterparty doesn't know who the sender is, signatures can be reused between senders, etc. It can only be required when actually verifying or signing, so you should do what we do with compliance data - throw if it isn't present.